ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -26,9 +26,10 @@ prefix_mask_packages: |
   # stick to GCC 9.x; using a too recent compiler in the compat layer complicates stuff in the software layer,
   # see for example https://github.com/EESSI/software-layer/issues/151
   >=sys-devel/gcc-10
-  # see https://github.com/EESSI/software-layer/issues/257
-  # and https://github.com/EESSI/software-layer/issues/258
-  <dev-libs/openssl-3
```

Comment:
`<` is incorrect, as that's masking (hiding) all OpenSSL versions < 3, while we actually want to do the exact opposite...

Reply:
ah yes, I hate gentoo